SWLOR.Game.Server.Scripts/Quest/WorkinForTheMan.cs

```diff
@@ -0,0 +1,17 @@
+﻿using SWLOR.Game.Server.Quest;
+
+namespace SWLOR.Game.Server.Scripts.Quest 
+{
+    public class WorkinForTheMan : AbstractQuest
+    {
+        public WorkinForTheMan()
+        {
+            CreateQuest(32, "Workin for the Man", "workin_for_man")
```

Comment:
You're gonna call me anal but can you put a single ' after "Workin"?

Reply:
Wants throwing axe instead of axe anal confirmed

Reply:
Changed